docs/guides/debugging-with-vscode.md

```diff
@@ -0,0 +1,88 @@
+# Debugging with vscode
+
+[Visual Studo Code](https://code.visualstudio.com/) is a light weight editor, with excellent debugging support. In this guide you'll be shown on how to get debugging working with `fuse-box` on both web apps and node applications. The source code for this example application can be found [here](/andrew-ross/fuse-debug).
```

Comment:
I think a full link is required, as we are hosting it
`/andrew-ross/fuse-debug`

Reply:
Done